nengo/neurons.py

```diff
@@ -463,6 +463,15 @@ def __init__(self, tau_rc=0.02, tau_ref=0.002, min_voltage=0, amplitude=1):
             tau_rc=tau_rc, tau_ref=tau_ref, amplitude=amplitude)
         self.min_voltage = min_voltage
 
+    @property
+    def _argreprs(self):
+        args = []
+        self._add_argrepr(args, 'tau_rc', 0.02)
+        self._add_argrepr(args, 'tau_ref', 0.002)
+        self._add_argrepr(args, 'min_voltage', 0)
```

Comment:
These three lines are copied from the parent `LIFRate`, we should use their method instead.

Reply:
I had thought about this, but the third line's actually different, and it's lines [1, 2, 4] that are copied. That is, `min_voltage` comes in between `tau_ref` and `amplitude`. I wanted to keep the ordering consistent with their respective constructors.